mido/midifiles/midifiles.py

```diff
@@ -399,18 +399,19 @@ def play(self, meta_messages=False):
 
         You will receive copies of the original messages, so you can
         safely modify them without ruining the tracks.
-
         """
-        sleep = time.sleep
-        now = time.time
-        t0 = now()
-        file_time = 0
+        t0 = time.time()
+        combined_time = 0.0
 
         for msg in self:
-            file_time += msg.time
-            duration_to_next_event = file_time - (now() - t0)
-            if (duration_to_next_event > 0):
-                sleep(duration_to_next_event)
+            combined_time += msg.time
+
+            # we compute the difference between the nominal combined time
```

Comment:
this comment doesn't seem of high value since it's reiterating the expression

I think the meaning of the expression is apparent, especially with the rename to `input_time` I suggested.  However, it could be clarified slightly with an intermediate variable for delta time (again avoiding a comment in favor of inherently descriptive code).
```
playback_time = time.time() - t0
duration_to_next_event = input_time - playback_time
```

Reply:
@belm0, I was struggling with this code until I saw your version here. This makes perfect sense to me. There's a `playback_time` that's trying to keep up with the `input_time` via `sleep()`.

`playback_time` is a very good name. Maybe the other variable could be `target_time`? I think `input_time` is fine though.